helm/helm-2to3-migration-app/values.yaml

```diff
@@ -0,0 +1,21 @@
+# Default values for helm-2to3-migration chart.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+name: helm-2to3-migration
+namespace: giantswarm
+
+userID: 1000
+groupID: 1000
+
+global:
+  image:
+    registry: quay.io
+
+image:
+  registry: quay.io
+  name: giantswarm/helm-2to3
+  tag: 1.0.2
```

Comment:
```suggestion
  tag: "[[ .Version ]]"
```

Reply:
Is that necessary? 
I think in `Chart.yaml` is enough. 
https://github.com/giantswarm/helm-2to3-migration-app/blob/ab1a70751122c62432c4ee400c331ac94856d43b/helm/helm-2to3-migration-app/Chart.yaml#L5

Reply:
Yes because the image is created by us.

e.g. https://github.com/giantswarm/cert-exporter/blob/e96954ca85e1c6bb790042f9230ad61519b837ac/helm/cert-exporter/values.yaml#L7

If its a retagged image then just the version is correct.

Reply:
I'm not sure how cert-exporter works in this case, but helm-2to3 should stay on the version among the list in here; https://quay.io/repository/giantswarm/helm-2to3?tab=tags

I think Just `1.0.x` would be fine, WDYT?

Reply:
No that won't work. The release process is the usual one for apps. So the version needs to be dynamic.

This is so when you push to test catalog it uses the version with the commit. e.g. `0.0.0-bff91bf578858bea5cfb8b57e26ca932677897ef`

https://github.com/giantswarm/giantswarm-playground-test-catalog/commit/30caa61e927bce235386bac7402ed46380cda14b

To release it you create a release tag like usual.

Reply:
OK I get this now with the image tags. Those tags are from archirect.

We should use `architect/push-to-docker` for Quay and Aliyun because we need the migration in China. Like we do here in chart-operator.

https://github.com/giantswarm/chart-operator/blob/847908f6b1a8dc6606df1facf19e57d6ab7870d1/.circleci/config.yml#L63-L87

Reply:
Now I know what you mean. 
In `cert-exporter` cases, we need to have the same tag version as app version since we upload a new version into quay. 
https://github.com/giantswarm/cert-exporter/blob/e96954ca85e1c6bb790042f9230ad61519b837ac/.circleci/config.yml#L118-L128

But this is not the case, `helm-2to3` image does not change at all. For similar cases, I think `cert-manager` is the best. 

https://github.com/giantswarm/cert-manager-app/blob/c63edb064aae9c7a091c0191b0eb2f1af215ae3f/helm/cert-manager-app/values.yaml#L11-L13

Reply:
> We should use architect/push-to-docker for Quay and Aliyun because we need the migration in China. Like we do here in chart-operator.

I think we should do that in `helm-2to3`, not this repo, WDYT?

Reply:
I think since this is an app we should use the usual release process like I described.

But I don't have more time to spend on this. So stopping here. Please make sure the image we use in China comes from Aliyun not Quay.

Reply:
This is more like nginx-ingress-controller, Do we have any app that have only retains Dockerfile? then we could merge this repo with https://github.com/giantswarm/helm-2to3

Reply:
We talked. I'd missed that the docker image in another repo. 🤦‍♂ 

So this approach is fine. 👍❤️